### PR TITLE
Chef registration

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "ignorePaths": ["node_modules", "dist", "public", "locales/de", "locales/fr", "locales/ru"],
-    "words": ["codegen", "esnext", "minifiable", "nextjs", "tailwindcss", "textsearch", "validatorjs"]
+    "words": ["codegen", "esnext", "minifiable", "nextjs", "tailwindcss", "Deutsch", "textsearch", "validatorjs"]
```
Comment (CemYil03) [inline .cspell.json:3]: Please not :)

File: src/components/cards/menuCard/PEMenuCard.tsx [hunk 2]
```diff
@@ -69,7 +69,13 @@ export default function PEMenuCard({
                 <div className="flex flex-row justify-between w-full">
                     <div className="flex flex-row items-center gap-2 overflow-hidden">
                         {chefProfilePictureUrl && (
-                            <img src={chefProfilePictureUrl} alt={chefProfilePictureUrl} className="rounded-4 w-6 h-6 object-cover" />
+                            <Image
+                                width={24}
+                                height={24}
+                                src={chefProfilePictureUrl ?? ''}
```
Comment (CemYil03) [inline src/components/cards/menuCard/PEMenuCard.tsx:75]: We shouldn't default th eimage url to an empty string, but instead not render the component at all.
{chefProfilePictureUrl && <Image .../>}
{!chefProfilePictureUrl && default thing}

File: src/components/cards/menuCard/PEMenuCardMobile.tsx [hunk 3]
```diff
@@ -45,7 +52,13 @@ export default function PEMenuCardMobile({
...
                 <div className="flex flex-row items-center gap-2 overflow-hidden">
                     {chefProfilePictureUrl && (
-                        <img src={chefProfilePictureUrl} alt={chefProfilePictureUrl} className="rounded-4 w-6 h-6 object-cover" />
+                        <Image
+                            width={24}
+                            height={24}
+                            src={chefProfilePictureUrl ?? ''}
+                            alt={chefProfilePictureUrl ?? 'menu card image'}
+                            className="rounded-4 w-6 h-6 object-cover"
+                        />
```
Comment (CemYil03) [inline src/components/cards/menuCard/PEMenuCardMobile.tsx:61]: Here you did it nicely 👌
Just one little improvement, chefProfilePictureUrl cannot be undefined in line 58 and 59, so you don't need the fallback

**Review by CemYil03 — CHANGES_REQUESTED** [review]
💬

File: .cspell.json (hunk 1 above)
Comment (Llavrov) [reply .cspell.json:3]: hah))

**Review by CemYil03 — APPROVED** [review]

**Pull request merged**